### PR TITLE
Fix noarch build

### PR DESCRIPTION
**Pull request by abloemert**
Checklist
* [x] Used a fork of the feedstock to propose changes
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/conda_smithy.html#how-to-re-render ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by abloemert** [comment]
@conda-forge-admin, please rerender

**Comment by daniel88ferreira** [comment]
@conda-forge/help-python can you please review?

**Comment by beckermr** [comment]
Thanks for the contribution! The staged recipes maintainers don't have commit rights on feedstocks. You will need to ask @conda-forge/inflection to merge this PR.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -13,7 +13,7 @@ source:
   {{ hash_type }}: {{ hash_val }}
 
 build:
-  number: 0
+  number: 1
   noarch: python
   script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
```
Comment (marcelotrevisani) [inline recipe/meta.yaml]:
```suggestion
  script: "{{ PYTHON }} -m pip install . -vv"
```

File: recipe/meta.yaml [hunk 2]
```diff
@@ -13,7 +13,7 @@ source:
   {{ hash_type }}: {{ hash_val }}
```
Comment (marcelotrevisani) [inline recipe/meta.yaml]:
```suggestion
  sha256: 18ea7fb7a7d152853386523def08736aa8c32636b047ade55f7578c4edeb16ca
```

Comment (marcelotrevisani) [reply recipe/meta.yaml]: and could you please remove the ``hash_type`` and ``hash_val`` ?

**Comment by daniel88ferreira** [comment]
@conda-forge/help-python can you please check now?
 @conda-forge/inflection please take a look and approve is review is positive.

**Comment by daniel88ferreira** [comment]
@pmlandwehr  Can you take a look?

**Comment by marcelotrevisani** [comment]
@conda-forge-admin , please rerender

**Comment by abloemert** [comment]
@ocefpaf @marcelotrevisani Thanks!

**Pull request merged**